radio/src/targets/common/arm/stm32/bootloader/boot.cpp

```diff
@@ -524,15 +524,12 @@ int main()
     }
 
     if (state == ST_REBOOT) {
-      lcdClear();
-      lcdRefresh();
-      lcdRefreshWait();
 
 #if defined(RTC_BACKUP_RAM)
       rtcInit();
       RTC->BKP0R = SOFTRESET_REQUEST;
 #endif
-
+      blExit();
```

Comment:
Since this is a common file... This needs either making conditional on the B&W radios, or implementations added for horus and nv14. After that this PR *should* start to build for all targets again. Since Raphael *did* mention blExit could be implemented differently for each target, I would suggest the latter.

```diff
diff --git a/radio/src/targets/horus/bootloader/boot_menu.cpp b/radio/src/targets/horus/bootloader/boot_menu.cpp
index 089bb3868..7a86981a9 100644
--- a/radio/src/targets/horus/bootloader/boot_menu.cpp
+++ b/radio/src/targets/horus/bootloader/boot_menu.cpp
@@ -235,3 +235,10 @@ void bootloaderDrawFilename(const char* str, uint8_t line, bool selected)
         lcd->drawSolidRect(119, 72 + (line * 25), 278, 26, 2, BL_SELECTED);
     }
 }
+
+void blExit()
+{
+      lcdClear();
+      lcdRefresh();
+      lcdRefreshWait();
+}
diff --git a/radio/src/targets/nv14/bootloader/boot_menu.cpp b/radio/src/targets/nv14/bootloader/boot_menu.cpp
index af6d49cee..4501dd034 100644
--- a/radio/src/targets/nv14/bootloader/boot_menu.cpp
+++ b/radio/src/targets/nv14/bootloader/boot_menu.cpp
@@ -240,3 +240,10 @@ void bootloaderDrawFilename(const char* str, uint8_t line, bool selected)
         lcd->drawSolidRect(DEFAULT_PADDING + 25, 72 + (line * 25), LCD_W - (DEFAULT_PADDING + 25) - 28, 26, 2, BL_SELECTED);
     }
 }
+
+void blExit()
+{
+      lcdClear();
+      lcdRefresh();
+      lcdRefreshWait();
+}
```

Reply:
ok thanks, then let's wait for his review